processes/ingest/chicago_isac.py

```diff
@@ -1,50 +1,14 @@
-import json
-import os
-import traceback
-import dataclasses
+from model import Source
+from services import ChicagoISACService
+from ..record_ingestor import RecordIngestor
 
-from mappings.chicago_isac import map_chicago_isac_record
-from managers import DBManager, S3Manager, WebpubManifest
-from model import Record, Part, FileFlags
-from logger import create_log
-from ..record_buffer import RecordBuffer
-from digital_assets import get_stored_file_url
 
-logger = create_log(__name__)
+class ChicagoISACProcess(RecordIngestor):
```

Comment:
Curious if you'd considered composition instead to reduce coupling? Like, the `ChicagoISACProcess` could *have* a record ingestor to do the ingest, that way if an ingest needs to do something weird with params or something, could be simpler. That said, the inheritance looks like it works well too!

Reply:
good callout - i suppose each source process are in fact record ingestors but composition i think works well because it's just a bit more explicit and readable.